.jenkins/library/vars/tests.groovy

```diff
@@ -98,11 +98,16 @@ def ACCContainerTest(String label, String version, List extra_cmake_args = []) {
                 cleanWs()
                 checkout scm
                 def cmakeArgs = helpers.CmakeArgs("RelWithDebInfo","OFF","ON","-DLVI_MITIGATION_BINDIR=/usr/local/lvi-mitigation/bin",extra_cmake_args.join(' '))
+                def hostOSVer = sh(
```

Comment:
Would it be possible to put `hostOSVer` in a helper function?

Reply:
Oh yeah, good suggestion!

Reply:
Doing so seems to introduce a bug. Will debug but may have to revert.